lib/huebsch.js

```diff
@@ -77,6 +77,27 @@ export const getFromHuebsch = async (body) => {
   }
 }
 
+const fetchWithRetry = async (url, retries = 3, waitForMs = 1000) => {
+  const debug = _debug('huebsch:fetchWithRetry')
```

Comment:
could we move the call to debug outside of `fetchWithRetry`? Otherwise we recreate the debug function every time this function is called